Covid19Radar/Tests/Covid19Radar.UnitTests/ViewModels/HelpPage/SendLogCompletePageViewModelTests.cs

```diff
@@ -81,6 +87,28 @@ public void OnClickSendMailCommandTests_Exception()
             Assert.Equal(1, actualCalls);
         }
 
+        [Fact]
+        public void OnCopyCommandTests()
+        {
+            var testLogId = "test-log-id";
+
+            var unitUnderTest = CreateViewModel();
+            unitUnderTest.Initialize(new NavigationParameters { { "logId", testLogId } });
+
+            string copyId = default;
+
+            unitUnderTest.CopyIdAsync = (LogId) =>
+            {
+                copyId = LogId;
+                return Task.CompletedTask;
+            };
+
+            unitUnderTest.OnCopyCommand.Execute(null);
+
+            mockUserDialogs.Verify(x => x.AlertAsync(It.IsAny<string>(), It.IsAny<string>(), "OK", null), Times.Never());
```

Comment:
テスト落ちたのがおかしいなと思ったのですが、ここが原因でした。

```suggestion
            mockUserDialogs.Verify(x => x.AlertAsync(It.IsAny<string>(), It.IsAny<string>(), "OK", null), Times.Once());
```

Reply:
ありがとうございます！